File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/MoreExercise/MatchTickets.java
package ConditionalStatementsAdvanced.MoreExercise;

import java.util.Scanner;

public class MatchTickets {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        double budget = Double.parseDouble(scan.nextLine());
        String category = scan.nextLine();
        int people = Integer.parseInt(scan.nextLine());
        double transportPrice = 0.0;
        double totalTicketPrice = 0.0;

        if (people < 5) {
            transportPrice = budget * 0.75;
        } else if (people < 10) {
            transportPrice = budget * 0.6;
        } else if (people < 25) {
            transportPrice = budget * 0.5;
        } else if (people < 50) {
            transportPrice = budget * 0.4;
        } else {
            transportPrice = budget * 0.25;
        }
        switch (category) {
            case "VIP":
                totalTicketPrice = people * 499.99;
                break;
            case "Normal":
                totalTicketPrice = people * 249.99;
                break;
        }
        if (budget >= totalTicketPrice + transportPrice) {
            System.out.printf("Yes! You have %.2f leva left.", budget - (totalTicketPrice + transportPrice));
        } else {
            System.out.printf("Not enough money! You need %.2f leva.", (totalTicketPrice + transportPrice) - budget);
        }
    }
}

File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/MatchTickets.js
function matchTickets(input) {

    let budget = Number(input[0]);
    let category = String(input[1]);
    let people = Number(input[2]);
    let transport = 0;
    let ticketPrice = 0;

    if (people > 0 && people <= 4) {
        transport = budget * 0.75;
    } else if (people >= 5 && people <= 9) {
        transport = budget * 0.6;
    } else if (people >= 10 && people <= 24) {
        transport = budget * 0.5;
    } else if (people >= 25 && people <= 49) {
        transport = budget * 0.4;
    } else if (people >= 50) {
        transport = budget * 0.25;
    }
    switch (category) {
        case "VIP": ticketPrice = 499.99 * people; break;
        case "Normal": ticketPrice = 249.99 * people; break;
    }

    if ((budget - transport) >= ticketPrice) {
        console.log(`Yes! You have ${(budget - transport - ticketPrice).toFixed(2)} leva left.`)
    } else {
        console.log(`Not enough money! You need ${(ticketPrice - (budget - transport)).toFixed(2)} leva.`)
    }
}
matchTickets(['30000', 'VIP', '49'])